graphql/schemabuilder/pagination.go

```diff
@@ -42,6 +42,14 @@ type ConnectionArgs struct {
 	Args   interface{}
 }
 
+// PaginationArgs are embedded in a struct
```

Comment:
why can't we expose `ConnectionArgs` directly?

Reply:
Connection args and pagination args are used differently. Connection args has an interface that is used to parse the user facing args whereas Pagination args are user facing (passed into the resolver).